users/tests/test_utils_functions.py

```diff
@@ -1,60 +1,56 @@
+            CustomValidationErrorText.VALIDATION_ERROR_EMAIL_MALFORMED_FIRST_PART, custom_exception.exception.message
         )
 
     def test_user_should_hold_email_with_valid_domain(self):
         email = "testuser@invaliddomain.com"
-        with self.assertRaises(CustomValidationError) as custom_exception:
-            users.common.utils.custom_validate_email_function(self, email)
+        with self.assertRaises(ValidationError) as custom_exception:
+            users.common.utils.custom_validate_email_function(email)
         self.assertEqual(
-            CustomValidationErrorText.VALIDATION_ERROR_EMAIL_MESSAGE_DOMAIN, str(custom_exception.exception)
+            CustomValidationErrorText.VALIDATION_ERROR_EMAIL_MESSAGE_DOMAIN, custom_exception.exception.message
         )
 
 
 class TestCustomSerializerEmailValidationFunction(TestCase):
     def test_user_email_should_hold_at_least_one_at_sign(self):
         email = "testusercodepoets.it"
-        with self.assertRaises(serializers.ValidationError) as serializer_exception:
-            users.common.utils.custom_validate_email_function(CustomRegisterSerializer(), email)
-        self.assertTrue(
-            CustomValidationErrorText.VALIDATION_ERROR_EMAIL_AT_SIGN_MESSAGE in str(serializer_exception.exception)
-        )
+        with self.assertRaises(ValidationError) as exception:
+            users.common.utils.custom_validate_email_function(email)
+        self.assertTrue(CustomValidationErrorText.VALIDATION_ERROR_EMAIL_AT_SIGN_MESSAGE in str(exception.exception))
 
     def test_user_email_should_not_hold_malformed_first_part(self):
         email = "@codepoets.it"
-        with self.assertRaises(serializers.ValidationError) as serializer_exception:
-            users.common.utils.custom_validate_email_function(CustomRegisterSerializer(), email)
+        with self.assertRaises(ValidationError) as exception:
+            users.common.utils.custom_validate_email_function(email)
         self.assertTrue(
-            CustomValidationErrorText.VALIDATION_ERROR_EMAIL_MALFORMED_FIRST_PART in str(serializer_exception.exception)
+            CustomValidationErrorText.VALIDATION_ERROR_EMAIL_MALFORMED_FIRST_PART in str(exception.exception)
         )
 
     def test_user_should_hold_email_with_valid_domain(self):
         email = "testuser@invaliddomain.com"
-        with self.assertRaises(serializers.ValidationError) as serializer_exception:
-            users.common.utils.custom_validate_email_function(CustomRegisterSerializer(), email)
+        with self.assertRaises(ValidationError) as exception:
+            users.common.utils.custom_validate_email_function(email)
         self.assertTrue(
-            CustomValidationErrorText.VALIDATION_ERROR_EMAIL_MESSAGE_DOMAIN_SHORT in str(serializer_exception.exception)
+            CustomValidationErrorText.VALIDATION_ERROR_EMAIL_MESSAGE_DOMAIN_SHORT in str(exception.exception)
```

Comment:
It is not `CustomValidationError` text anymore.

Reply:
done

Reply:
This change makes changes in many files, I think that I should change commit name after rebase, is that right @rwrzesien ?

Reply:
If you think this change is not really related to this commit, go ahead and create new one for it.